### DISCUSSION TITLE
Monailabel- Pancreas Task 07

### DISCUSSION THREAD

**Discussion by vincentochs** [Project-MONAI/MONAILabel#1147] [General]
Hello together :)

I found various implementations for the Spleen - but is there an already pre-trained dataset and a respective model for the pancreas task 07 existing?

Maybe with tumor?

Thank you very much for any information!

**Comment by diazandr3s** [comment]
Hi @vincentochs,

The available implementations are references for the users to develop on top of that or make incremental changes to them.
Currently, there is no pretrained model for pancreas segmentation. But it shouldn't be difficult for you to create one using the [Task07_Pancreas](https://drive.google.com/drive/folders/1HqEgzS8BV2c7xYNrZdEAnrHk7osJJ--2) dataset from the medical segmentation decathlon and/or the [BTCV dataset](https://www.synapse.org/#!Synapse:syn3193805/wiki/217789).

Hope this helps,

**Reply by vincentochs** [reply]
Thank you
